configure.py

```diff
@@ -756,7 +756,7 @@ def find_ninja():
                         help='List all available build artifacts, that can be passed to --with')
 arg_parser.add_argument('--date-stamp', dest='date_stamp', type=str,
                         help='Set datestamp for SCYLLA-VERSION-GEN')
-arg_parser.add_argument('--use-cmake', action='store_true', help='Use CMake as the build system')
+arg_parser.add_argument('--use-cmake', action=argparse.BooleanOptionalAction, default=False, help='Whether to use CMake as the build system')
```

Comment:
Can you please clarify how does `--no-use-cmake` does not use cmake when `--use-cmake` is defaulted to `false`?

Reply:
@regevran did you get a chance to read the commit message? i am happy to explain it in more details if you have a minute.